src/app/export-csv/csv-export-sample-1/csv-export-sample-1.component.ts

```diff
@@ -0,0 +1,43 @@
+import { Component, ViewChild } from '@angular/core';
+
+import { IgxCsvExporterService,
+         IgxCsvExporterOptions,
+         IColumnExportingEventArgs,
+         IRowExportingEventArgs,
+         ICsvExportEndedEventArgs,
+         CsvFileTypes } from "igniteui-angular/services/index";
+
+import { IgxGridComponent } from "igniteui-angular/grid/grid.component";
+
+@Component({
+  selector: 'app-csv-export-sample-1',
+  templateUrl: './csv-export-sample-1.component.html',
+  styleUrls: ['./csv-export-sample-1.component.scss']
+})
+export class CsvExportSample1Component {
+
+  @ViewChild("igxGrid1") igxGrid1: IgxGridComponent;
+  localData = [
+    { Name: "Eric Ridley", Age: "26" },
+    { Name: "Alanis Brook", Age: "22" },
+    { Name: "Jonathan Morris", Age: "23" }
+  ];
+
+  constructor() {
+  }
+
+  exportButtonHandler() {
+    let ees: IgxCsvExporterService = new IgxCsvExporterService();
```

Comment:
@rmkrmk: Don't create services directly. Instead add them to the providers array of the module/component and inject them in the constructor. The angular injector will handle the creation for you.